pkg/exporters/http_exporter.go

```diff
@@ -297,7 +297,7 @@ func (e *HTTPExporter) sendAlertLimitReached(ctx context.Context) error {
 
 	logger.L().Error("Alert limit reached",
 		helpers.Int("alerts", e.alertMetrics.count),
-		helpers.String("since", e.alertMetrics.startTime.Format(time.RFC3339)))
+		helpers.String("since", e.alertMetrics.startTime.Format(time.RFC3339))) // TODO check with Amit
```

Comment:
What is the question?

Reply:
should we log an error here or is a debug enough? what can the user do about it?

Reply:
This Error is for us. We can switch it to Warning maybe?